libs/ui/src/admin/RecommendedTweetCard/RecommendedSocialButtons.tsx

```diff
@@ -60,6 +60,10 @@ export const RecommendedSocialButtons: FC<RecommendedSocialButtonsProps> = ({
     .join(' ')
   const quoteTweet = [tweet?.text, mentions].filter(a => !!a).join('\n\n')
 
+  const tweetImageUrl = tweet?.image
+    ? ''
+    : `https://twitter.com/${tweet.originalTweet?.user?.username}/status/${tweet?.originalTweet?.id}/video/1`
```

Comment:
If users create new tweets by uploading images, we will have images from Strapi without having an original tweet.

If there is no original tweet, the url will be `https://twitter.com/undefined/status/undefined/video/1`. I think we should't generate url if there is no original tweet.

```suggestion
  const tweetImageUrl = tweet?.image && tweet.originalTweet?.user && `https://twitter.com/${tweet.originalTweet?.user?.username}/status/${tweet?.originalTweet?.id}/video/1`
```

Reply:
Or maybe its better to check only if original tweet has an image or video?

const tweetImageUrl = tweet?.originalTweet?.image || tweet?.originalTweet?.video && `https://....`